src/MaxText/examples/grpo_llama3_1_70b_demo_pw.py

```diff
@@ -141,18 +141,21 @@
 TRAIN_FRACTION = 1.0
 
 
+# ====== Checkpoint directory =====
+MODEL_CKPT_PATH = "/path/to/model/ckpt"
+
 # ====== Checkpoint directory =====
 LOG_DIR = f"{HOME}/content/tensorboard/grpo/logs_llama3/"
 if not os.path.exists(LOG_DIR):
   os.makedirs(LOG_DIR)
 
 # ===== Profiling =====
-PROFILE_DIR = f"gs://mazumdera-test-bucket-europe-west4/rl-tuning/grpo/anisha-{run_id}/profiles_llama3/"
+PROFILE_DIR = f"/path/to/profile_dir/{run_id}/profiles_llama3/"
 if not epath.Path(PROFILE_DIR).exists():
   epath.Path(PROFILE_DIR).mkdir(parents=True)
 
 # ====== Checkpoint saving ======
-CKPT_DIR = f"gs://mazumdera-test-bucket-europe-west4/rl-tuning/grpo/anisha-{run_id}/ckpts_llama3/"
+CKPT_DIR = f"/path/to/ckpt_dir/{run_id}/ckpts_llama3/"
```

Comment:
Nit: if you are pulling `MODEL_CKPT_PATH ` to a separate variable, you could do the same for the others. Not a big deal either way though